bluesky/callbacks/broker.py

```diff
@@ -36,7 +36,7 @@ def descriptor(self, doc):
     def event(self, doc):
         # the subset of self.fields that are (1) in the doc and (2) unfilled
         fields = [field for field in self.fields
-                  if (doc['data'].get(field) and
```

Comment:
Does `fill_events` just ignore fields that an event doesn't have? Anyway, this seems safer, cleaner.